### DISCUSSION TITLE
Is it a typo?

### DISCUSSION THREAD

**Discussion by dwangF0** [lucidrains/audiolm-pytorch#232] [Q&A]
In the main branch, line 671 of this file ./audiolm_pytorch/audiolm_pytorch.py, 

line 671 "return logits, new_kv_cache"

Is it supposed to be  "return logits, kv_cache"? I searched the whole forward function, and cannot find new_kv_cache, but only kv_cache. 

Does anyone have any idea of this?

**Comment by lucidrains** [comment]
oops yea, should be fixed

**Reply by dwangF0** [reply]
no worry, thanks for the quick response!
